lib/full-site-editing/edit-site-page.php

```diff
@@ -170,6 +170,8 @@ function gutenberg_edit_site_init( $hook ) {
 	wp_enqueue_script( 'wp-format-library' );
 	wp_enqueue_style( 'wp-edit-site' );
 	wp_enqueue_style( 'wp-format-library' );
+	wp_enqueue_style( 'wp-block-library-theme' );
```

Comment:
I thought we only enqueued these if the theme opted-in into them ([related](https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#default-block-styles)), so I'm not sure whether we should load these conditionally.

Reply:
Yes, I think we should probably load these conditionally. I think @scruffian was working on a related PR lately, not sure if it was for the site editor or post editor.

Reply:
I think you may be referring to https://github.com/WordPress/gutenberg/pull/29252 Pushed the same logic here: these styles are loaded if the theme declares support for them or if there's no editor styles.